### ISSUE TITLE
Submit button in create/invite party modal has no label.

### ISSUE THREAD

**Issue by beejunk** [HabitRPG/habitica#8391]
[//]: # (Before logging this issue, look through common problems at https://github.com/HabitRPG/habitrpg/issues If you find your issue there, read at least the first post to see if there is a workaround for you)

[//]: # (Github is primarily used for reporting bugs. If you have a feature request, use "Help > Request a Feature" so that the feature request can be vetted by the larger Habitica community)

[//]: # (To report a bug in one of the mobile apps, please report it in the correct repository. Android: https://github.com/HabitRPG/habitrpg-android, iOS: https://github.com/HabitRPG/habitrpg-ios)

[//]: # (For more guidelines see https://github.com/HabitRPG/habitrpg/issues/2760)

[//]: # (Fill out relevant information - UUID is found in Settings -> API)
General Info
  * UUID: a654bd1f-e925-438b-8d98-9ac4462b973f
  * Browser: Firefox 50
  * OS: Ubuntu Linux

### Description
[//]: # (Describe bug in detail here. Include pictures if helpful.)

The submit button in the create/invite party modal has no label. This causes it to look like a blue strip of unknown purpose. See this image for an example:

![party_submit_issue](https://cloud.githubusercontent.com/assets/798619/21821722/36a38f06-d73a-11e6-8ef0-a25496965816.png)

This also happens in Chromium 55 on Ubuntu.

**Comment by beejunk** [comment]
This appears to happen specifically when clicking the 'Battle Monsters With Freinds' button at the top of the page. The 'Invite Friends' button on the Party page works correctly.

**Comment by eanplatter** [comment]
Same thing on Chrome / OSX

**Comment by librarianmage** [comment]
I can take a look!

**Comment by eanplatter** [comment]
I might also look at it, I've never delved into this codebase before :P

**Comment by librarianmage** [comment]
@eanplatter Then I'll let you take a peek, as this doesn't look too challenging. Best of luck!

**Comment by eanplatter** [comment]
Thanks! @MathWhiz!

**Comment by eanplatter** [comment]
Found the regression! Here: https://github.com/HabitRPG/habitica/pull/8328

Looks like the default text isn't being applied to `group.sendInviteText` from the controller, I am digging into why!

**Comment by eanplatter** [comment]
Cool, I think I figured it out.

When #8328 was added, the text for the buttons was changed from `=env.t('sendInvitations')` to the dynamic message influenced by the `groupsCtrl` `{{group.sendInviteText}}` which makes sense to show the different button text.

The only problem is that if the button isn't subject to the fee text, then the `group` has no concept of `sendInviteText`.

All I did was went to where the `group` object originates `groupService.js` and added a default sendInviteText. I will make a PR here shortly!
